modules/post/multi/manage/shell_to_meterpreter.rb

```diff
@@ -53,6 +57,9 @@ def initialize(info = {})
     deregister_options('PERSIST', 'PSH_OLD_METHOD', 'RUN_WOW64')
   end
 
+  VALID_PSH_ARCH_OVERRIDE = ['x64', 'x86']
+  VALID_PLATFORM_OVERRIDE = ['windows', 'win', 'osx', 'solaris', 'python', 'linux', 'netware', 'android', 'java', 'r', 'ruby', 'cisco', 'juniper', 'unifi', 'brocade', 'mikrotik', 'artista', 'bsd', 'aix', 'hpux', 'irix', 'unix', 'php', 'js', 'firefox', 'nodejs', 'mainframe', 'multi', 'hardware', 'apple_ios']
```

Comment:
Is this list coming from somewhere else that we can just depend on? 👀

Reply:
Didn't see one on first pass but I'll take another look. Maybe can sort through platform subclasses